src/EditModel/event-program/notifications/selectors.js

```diff
@@ -1,6 +1,15 @@
 import { get, compose, first, __ } from 'lodash/fp';
 
-export const getStageNotifications = ({ programStages, programStageNotifications }) => compose(get(__, programStageNotifications), get('id'), first)(programStages);
+export const getStageNotifications = ({ programStages, programStageNotifications }) =>
+    compose(get(__, programStageNotifications), get('id'), first)(programStages);
+
+export const getProgramStageDataElements = ({ programStages, availableDataElements }) => {
+    const programStageDataElements = programStages[0].programStageDataElements
```

Comment:
Will it always be the first element of the programStages array?

Reply:
Yes, because event programs always have 1 stage (which is hidden in the UI).